src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,17 @@
+.catalog {
+  &__cards {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 50px 40px 73px;
+    width: 1024px; // 4 * 200 = 800; 4 * 48 = 192 800 + 192 = 992
+    margin: auto;
+  }
+
+  &__card:not(:nth-of-type(4n)) {
```

Comment:
10 item of checklist
are you sure you need these variables that are used only ones?